### PR TITLE
Updated c++std for bdb, boost, qt and zeromq

### PR DESCRIPTION
**Pull request by mxaddict**
This PR is the replacement for #890

### CODE REVIEW

**Comment by navbuilder** [comment]
A new build of 5cadea05d93ec138c85b841443b27399d46aad40 has completed succesfully!
Binaries available at https://build.nav.community/binaries/update-cppstd-for-some-depends-packages

**Review by chasingkirkjufell — APPROVED** [review]
tested basic wallet functions with gitian build on windows and mainnet syncning

**Comment by mxaddict** [comment]
@aguycalled I think this PR is ready for review as well

File: depends/packages/qt.mk [hunk 1]
```diff
@@ -30,7 +30,7 @@ define $(package)_set_vars
 $(package)_config_opts_release = -release
 $(package)_config_opts_debug = -debug
 $(package)_config_opts += -bindir $(build_prefix)/bin
-$(package)_config_opts += -c++std c++11
+$(package)_config_opts += -c++std c++1z
```
Comment (aguycalled) [inline depends/packages/qt.mk:33]: is this a type?

Comment (mxaddict) [reply depends/packages/qt.mk:33]: No, it's from upstream

**Review by aguycalled — APPROVED** [review]

**Pull request merged**